Websites/perf.webkit.org/tools/sync-commits.py

```diff
@@ -287,7 +287,8 @@ def _find_next_hash(self, hash_to_find):
 
     def _fetch_remote(self):
         if self._report_svn_revision:
-            self._run_git_command(['pull'])
+            self._run_git_command(['fetch', 'origin', 'main'])
```

Comment:
Can we make the assuming it's always `main` or we should use `self. _git_branch` if it's not None?

Reply:
@JonWBedard I think we can, right ?

Reply:
Maybe we want this script support other repo I guess

Reply:
WebKit will always use `main`. Generally I've dealt with this by declaring a `DEFAULT_BRANCH`, either globally or passed into a constructor